pkg/arvo/app/chat.hoon

```diff
@@ -225,109 +146,70 @@
     (mean [leaf+"invalid wire for diff: {(spud wir)}"]~)
   ?+  i.wir
     (mean [leaf+"invalid wire for diff: {(spud wir)}"]~)
-    ::
-    ::  %circles wire
-    ::
+  ::
       %circles
     ?>  ?=(%circles -.piz)
-      =/  str  %=  str.sta
-        circles  cis.piz
-      ==
-      :-  (send-chat-update [[%circles cis.piz] str])
-      this(str.sta str)
+    =.  str  str(circles cis.piz)
+    :_  this(str str)
+    (send-chat-update [[%circles cis.piz] str])
+  ::
+      %circle
+    ?>  ?=(%circle -.piz)
+    =/  circle=circle:hall  [our.bol &3:wir]
```

Comment:
Can we please just to `i.t.wir` or whatever for these? Might want to do a `=*` either case, to make it clear what it's actually pointing to.